modules/virt-configuring-ssh.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-configuring-ssh.adoc
+
+:_content-type: PROCEDURE
+[id="virt-configuring-ssh_{context}"]
+= Configuring SSH for quick virtual machine access
+
+You can configure your local SSH client to quickly connect to your virtual machine (VM). This configuration makes it possible to leverage `virtctl` to relay an SSH command to the target VM.
+
+.Prerequisites
+* You have installed the OpenShift CLI (`oc`).
+* You have installed the `virtctl` client.
+* The virtual machine you want to access is running.
+
+.Procedure
+
+. Modify the `~/.ssh/config` on your client machine to include the following text:
++
+[source,terminal]
+----
+Host *.cluster
+ProxyCommand virtctl port-forward --stdio=true %h %p

```

Comment:
I think this line needs to be indented.
 @geetikakay WDYT?

Reply:
+1 @apinnick you are right. Thanks for an additional eye on it